examples/features/grid-spatial-filter.js

```diff
@@ -0,0 +1,292 @@
+/* global FeatureGridWithSpatialFilter */
+Ext.require([
+    'Ext.container.Container',
+    'Ext.panel.Panel',
+    'Ext.grid.Panel',
+    'Ext.grid.filters.Filters',
+    'GeoExt.component.Map',
+    'GeoExt.data.store.Features',
+    'GeoExt.util.OGCFilter'
+]);
+
+var olMap;
+var gridWest;
+var featStore;
+var wmsLayer;
+var drawSelectPolygonInteraction;
+var activeFilters = [];
+var spatialOperators = Ext.create('Ext.data.Store', {
+    fields: ['abbreviation', 'name'],
+    data: [{
+        abbreviation: 'intersect',
```

Comment:
Not really a `abbreviation`, or? More something like `value`.

Reply:
Absolutely, renamed in b0245d6